src/bot/utils.py

```diff
@@ -0,0 +1,22 @@
+from telegram import Update
+
+
+def delete_previous(coroutine):
+    """Для функций, отправляющих сообщения с inline-кнопками.
+    Удаляет сообщение с кнопками, приведшее к вызову функции."""
+
+    async def wrapper(*args, **kwargs):
+        result = await coroutine(*args, **kwargs)
```

Comment:
```suggestion
        result = await coroutine(update: Update, *args, **kwargs)
```
Не нужно тут использовать цикл...